### ISSUE TITLE
On keyboard navigation - cannot read property 'nativeElement' of undefined is thrown

### ISSUE THREAD

**Issue by zdrawku** [IgniteUI/igniteui-angular#1382]
### Description  
Error is thrown on keyboard navigation. This issue is not reproducible with **5.3.x**.

Also in **5.3.x** the performance is **very smooth**, compared to the current version.

I've tracked down the issue to method _**focusNextCell** in the cell component.

### Steps to reproduce  

1. Open **demos/app/grid-column-resizing** sample.
2. Start to navigate through the cells by pressing or long-pressing the arrow keys.

### Result  
Error is thrown

### Expected result  
Error should not be thrown

### Attachments  
![fbmsgbot](https://user-images.githubusercontent.com/11193764/40420364-e622f5fa-5e90-11e8-9d6e-fad7881857a5.gif)
